en/faq/external-resources.md

```diff
@@ -5,6 +5,10 @@ description: How to use external resources with Nuxt.js?
 
 ## Global Settings
 
+You can include your external resources in the head object or function.
+The following example is with an object, the second one is with a function. 
+If you encounter problems you may would like to switch to the head function.
```

Comment:
Especially when you want to use values coming from the Vue component (so all "non-static" content), a function is the only way to go (and to ensure dependencies are tracked and reactivity works).

It'd be nice if you could mention that somewhere :relaxed: